src/locales/default.json

```diff
@@ -422,6 +422,12 @@
   "goToMigratedPool": "Go to migrated pool",
   "governance": "Governance",
   "grants": "Grants",
+  "greatMigration": {
+    "warning": {
+      "title": "Move your liquidity for future BAL incentives",
+      "text": "Liquidity mining incentives have expired for this pool, since upgraded pools are now live. LPs are encouraged to unstake existing positions, withdraw their liquidity and add it to an incentivized pool."
```

Comment:
This is technically inaccurate atm because the gauges haven't been killed yet

Reply:
Gauges are expected to be killed on [March 7th](https://forum.balancer.fi/t/bip-177-great-migration-wave-1/4364)

Reply:
cc @pkattera

Reply:
Also the gauges for the new pools can already be voted for but aren't getting many votes yet. Maybe we wait to merge this next Wednesday after there're more clarity on where incentives will land?